lib/ansible/modules/monitoring/zabbix/zabbix_host_triggers.py

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) stephane.travassac@fr.clara.net
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+RETURN = '''
+---
+output:
+    description: Host Zabbix Triggers in Json format
+    returned: On success
+    type: complex
+    contains:
+            triggers_ok:
+                - comments: Raise an alert whenever there is no data received from the agent during
```

Comment:
Please specify the necessary fields for each return value as described in the [documentation](https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#return-block), and maybe move all the fields under `triggers_ok` and `triggers_problem` to their corresponding `sample` field.